datahub-web-react/src/app/shared/tags/TagProfileDrawer.tsx

```diff
@@ -0,0 +1,417 @@
+    const entityQuery = `tags:"${data?.tag?.properties?.name}"`;
+    const ownersEmpty = !data?.tag?.ownership?.owners?.length;
+    const description = data?.tag?.properties?.description || '';
+    const hexColor = data?.tag?.properties?.colorHex || '';
+
+    const [showAddModal, setShowAddModal] = useState(false);
+    const [showEditModal, setShowEditModal] = useState(false);
+    const [updatedDescription, setUpdatedDescription] = useState('');
+    const overLimit = removeMarkdown(updatedDescription).length > 80;
+    const [expanded, setExpanded] = useState(!overLimit);
+    const [displayColorPicker, setDisplayColorPicker] = useState(false);
+    const [colorValue, setColorValue] = useState('');
+
+    useEffect(() => {
+        setUpdatedDescription(description);
+        // Manual Color Set-Up, Need to set once it will come from backend
+        setColorValue(hexColor);
+    }, [description, hexColor]);
+
+    const allSearchResultsByType = useGetAllEntitySearchResults({
+        query: entityAndSchemaQuery,
+        start: 0,
+        count: 1,
+        filters: [],
+    });
+
+    const statsLoading = Object.keys(allSearchResultsByType).some((type) => {
+        return allSearchResultsByType[type].loading;
+    });
+
+    const someStats =
+        !statsLoading &&
+        Object.keys(allSearchResultsByType).some((type) => {
+            return allSearchResultsByType[type]?.data?.search.total > 0;
+        });
+
+    if (error || (!loading && !error && !data)) {
+        return <Alert type="error" message={error?.message || 'Entity failed to load'} />;
+    }
+
+    // Handle Color Picker Change
+    const closePicker = async () => {
+        if (displayColorPicker) {
+            message.loading({ content: 'Saving...' });
+            try {
+                await setTagColorMutation({
+                    variables: {
+                        urn,
+                        colorHex: colorValue,
+                    },
+                });
+                message.destroy();
+                message.success({ content: 'Color Updated', duration: 2 });
+                setDisplayColorPicker(false);
+            } catch (e: unknown) {
+                message.destroy();
+                if (e instanceof Error) {
+                    message.error({ content: `Failed to update Color: \n ${e.message || ''}`, duration: 2 });
+                }
+            }
+            refetch?.();
+        }
+    };
+
+    const handlePickerClick = () => {
+        setDisplayColorPicker(!displayColorPicker);
+        closePicker();
+    };
+
+    const handleColorChange = (color: any) => {
+        setColorValue(color?.hex);
+    };
+
+    const EditButton = (
+        <>
+            <EditIcon twoToneColor="#52c41a" onClick={() => setShowEditModal(true)} />
+            <EditDescriptionText> Edit Description</EditDescriptionText>
+        </>
```

Comment:
lets make the text clickable as well

Reply:
Removed the text as suggested

Reply:
[https://github.com/linkedin/datahub/pull/4132/commits/f314d27106cd3135e7def6ac9db381a764eb4d73](url)